### DISCUSSION TITLE
How to handle background jobs?

### DISCUSSION THREAD

**Discussion by adamzerner** [freshframework/fresh#1993] [Q&A] [answered]
What options are available and recommended for having background jobs in a Fresh app? I'm looking for something similar to Ruby on Rails' [Active Job](https://guides.rubyonrails.org/active_job_basics.html).

In my particular [use case](https://filterhn.com/), I want to hit the Hacker News API every five minutes, process the data, and save stuff to my database.

**Comment by adamzerner** [comment]
[deno_cron](https://deno.land/x/deno_cron@v1.0.0) and [croner](https://deno.land/x/croner@7.0.3) both look like options.

**Comment by waptik** [comment, marked as answer]
There's also the [first class support for cron](https://deno.com/blog/cron). Though it's still unstable and lacks some features but it's enough for basic cron jobs
